core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/SdkClientOption.java

```diff
@@ -51,6 +52,12 @@
      */
     public static final SdkClientOption<RetryPolicy> RETRY_POLICY = new SdkClientOption<>(RetryPolicy.class);
 
+    /**
+     * @see ClientOverrideConfiguration#retryStrategy()
+     */
+    @SuppressWarnings("rawtypes")
```

Comment:
Why is this suppression added?

Reply:
I'm using the rawtype `RetryStrategy` instead of `RetryStrategy<B, T>`

Reply:
Just to clarify, I cannot use the type params here, even using the generic with wildcards will make the compiler to protest.